integration_test/testdata/launcher-static.yml

```diff
@@ -1,5 +1,6 @@
 configType: java
 configVersion: 1
+serviceName: primary
```

Comment:
If this isn't a requirement, then in some ways I would prefer we have a test where it is blank, since that's more likely to have issues.

Reply:
This belongs in the config parsing testing - since what we want to test is that if we leave it off, we still get a valid string back from the parsing. Added some assertions to the config testing for this.